howso/direct/client.py

```diff
@@ -717,8 +738,10 @@ def is_match(name):
         for _, instance in self.trainee_cache.trainees():
             if is_match(instance.name):
                 trainees.append(
-                    TraineeIdentity(
-                        name=instance.name, id=instance.id)
+                    {
+                        "name": instance.name,
+                        "id": instance.id
+                    }
                 )
```

Comment:
`instance` should now be a dict here. will need to update the accessors for name and id.

Reply:
Addressed by: 5717250